themes/src/main/resources/theme/keycloak.v2/account/messages/messages_en.properties

```diff
@@ -95,6 +96,9 @@ webauthn-passwordless-display-name=Security Key
 webauthn-passwordless-help-text=Use your security key for passwordless log in.
 basic-authentication=Basic Authentication
 invalidRequestMessage=Invalid Request
+successSetDefaultMessage={0} was set as default.
+setDefaultCredTitle=Set {0} as default
+setDefaultCred=Set {0} as default authenticator application?
```

Comment:
This implies that this PR is just for set default credential of type "authenticator application" . I think it will be good if this PR is not specific only to "Authenticator application" (OTP) credential type, but can be used for all other credential types of all factors.